apps/air-discount-scheme/web/public/mockServiceWorker.js

```diff
@@ -0,0 +1,235 @@
+/**
```

Comment:
Does this need to be duplicated between projects?

Reply:
I used the documentation from here: https://docs.devland.is/handbook/libs/api/mocks#usage-in-new-project

If this can somehow be simplified that would be cool, but preferably as an improvement later.